api/types/sso_test.go

```diff
@@ -0,0 +1,127 @@
+package types
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestSSODiagnosticInfo_GetValue(t *testing.T) {
+	mkInfo := func(value interface{}) *SSODiagnosticInfo {
+		info, err := NewSSODiagnosticInfo(SSOInfoType_UNKNOWN, value)
+		require.NoError(t, err)
+		return info
+	}
+
+	type errInfo struct {
+		Message string
+		Error   string
+	}
+
+	tests := []struct {
```

Comment:
add a test case that results in an error?

Reply:
Done.